tests/unit/mainwindow/test_prompt.py

```diff
@@ -53,26 +53,26 @@ def _get_prompt_func(path):
         (2, 'next', 'a'),
         (2, 'prev', 'b'),
     ])
-    def test_simple_completion(self, tmpdir, get_prompt, steps, where,
+    def test_simple_completion(self, tmp_path, get_prompt, steps, where,
                                subfolder):
         """Simply trying to tab through items."""
-        testdir = tmpdir / 'test'
+        testdir = tmp_path / 'test'
         for directory in 'abc':
-            (testdir / directory).ensure(dir=True)
+            (testdir / directory).mkdir(exist_ok=True, parents=True)
```

Comment:
Is the `exist_ok=True` needed here? We should get an empty `tmp_path` for every time this test runs, so I don't think it should ever exist?